src/Acuminator/Acuminator.Tests/Tests/Utilities/NestedInvocationWalker/NestedInvocationWalkerTests.cs

```diff
@@ -159,7 +165,7 @@ public async Task Constructor(string text)
 			walker.Locations.Should().BeEquivalentTo((Line: 13, Column: 14));
 		}
 
-		[Theory]
+		[Theory(Skip = "The test is incorrect for current implementation of nested walker which does not step inside lambdas")]
 		[EmbeddedFileData("LocalLambda.cs")]
```

Comment:
@dropsonic please check out this unit test, it is incorrectly written. The exception walker, as it is implemented now, will never visit any nodes in lambda expressions because the base nested walker skips them

Reply:
If it is incorrect, why did it pass before? It requires a valid location from the nested walker.

Reply:
@dropsonic  that's a very interesting question. Actually the syntax tree created by the old version of Roslyn is quite strange:
--Class
  --Method
     --Body
         1. LocalDeclarationStatement
         2. ThrowStatement
         3. ExpressionStatement

The Syntax tree in the new Roslyn however looks like this:
[(https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzhgHwAEAmARgFgAoIgBgAIiyUBuauxsgOgBkBLAHYBHNjQZMuAFRgIMojkwCso6gOwBbGLgAO2MDHoBBMAFd1g7BmjUA3tQCQRAMyMS9AGIQIDu1XuOXIhR6ABEIAGUITQwAC0EAcwAKAEoHe19/ewA3bCh6PQx6AF56FOKAPnpYqAgAd3oBGHqAUQR9bQw+CAEU0UyC3rSAXwcRqiGgA===)]

--Class
  --Method
     --Body
         1. LocalDeclarationStatement
         2. ExpressionStatement

I suspect it is caused by the inroduction of the throw expressions to C# 7
You can see now the reason why the test passed